packages/layouts/src/components/SplitLayout/index.tsx

```diff
@@ -1,41 +1,39 @@
 import React from 'react';
-import { Omit } from 'utility-types';
+import { elementType } from 'airbnb-prop-types';
 import withStyles, { WithStylesProps } from '@airbnb/lunar/lib/composers/withStyles';
+import Aside from '../Aside';
 import Layout, { Props as LayoutProps, AsideProps } from '../Layout';
 
-export type Props = Required<AsideProps> &
-  Omit<LayoutProps, 'children' | 'fluid' | 'noBackground' | 'noPadding'>;
+export type Props = Required<AsideProps> & Pick<LayoutProps, 'fluid'>;
 
-/** A symmetrical two-column layout with optional top and side navigation. */
+/** A symmetrical two-column layout. */
 export class SplitLayout extends React.Component<Props & WithStylesProps> {
+  static propTypes = {
+    after: elementType(Aside).isRequired,
+    before: elementType(Aside).isRequired,
+  };
+
   render() {
-    const { cx, before, after, styles, ...props } = this.props;
+    const { after, before, cx, fluid, styles } = this.props;
 
     return (
-      <Layout {...props} noBackground noPadding>
+      <Layout noBackground noPadding fluid={fluid}>
         <div className={cx(styles.wrapper)}>
           <div className={cx(styles.column)}>{before}</div>
-          <div className={cx(styles.column, styles.column_after)}>{after}</div>
+          <div className={cx(styles.column)}>{after}</div>
         </div>
       </Layout>
     );
   }
 }
 
-export default withStyles(({ unit }) => ({
+export default withStyles(() => ({
   wrapper: {
     display: 'flex',
     height: '100%',
   },
 
   column: {
     width: '50%',
-    flexGrow: 0,
-    flexShrink: 0,
-    padding: unit * 2,
```

Comment:
add column padding back and `noPadding` prop? that'd keep layout gutters consistent

Reply:
or did the padding move into the before/after components? checking

Reply:
it did! resolving myself